ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -328,6 +330,28 @@ def ocs_install_verification(
         log.info("Verified - pool crush rule is with type: zone")
 
 
+def check_max_osds_per_node(max_number_osds=2):
```

Comment:
In production clusters, we can support 3 OSDs per node
In non-production clusters of ocs-ci, we can support <3. So, how do we handle this ?